### ISSUE TITLE
Firefox/NVDA: Unable to Read Content of Controls since Update

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#8134]
STR:

1. Go to the Google homepage (http://www.google.com/)

2. Navigate to the "Search" combo box

3. Notice that when attempting to read by character in Browse Mode that this is no longer possible; NVDA treats the control as a single character

Expected:

Previously, it was possible to read character-by-character; this seems to now not be possible with several types of controls (including both edit fields and combo boxes). Another example where this can be observed can be found on the main Mozilla homepage, where there is a field labeled "Your Email Here"; interestingly, it's no longer possible to even do a "Read Current Line" on these if you wish to spell the content.

I originally noticed this in October last year on both Windows 7/10; I'm running Firefox 59.0.2 and the latest Next snapshot.

**Comment by Brian1Gaff** [comment]
Well,  the latest next is doing OK in Firefox 51/2 here OK.
 Thus it could b something related to the new Quantum, but I'd have thought 
something as serious as that would have shown up by now?
 How does the release version and master snap cope in the same 
circumstances?

**Comment by tapper82** [comment]
Hi just tested this.
Windows 10 Version 1803 (OS Build 17133.1) 64 bit.
Firefox
Version 
61.0a1
Build ID 
20180401220058
(NVDA) Version: next-14987,359dd795

I am sorry but I cant reproduce this bug.
Pleas try updating to a later build of Firefox.

**Comment by zahra21** [comment]
hi.
i am not familiar with firefox quantum.
but did you test with javascript desabled?
i believe that activating javascript is the source of many vulnerability and security problems as well as accessibility issues.

**Comment by jcsteh** [comment]
[**@elliott94**](https://github.com/elliott94) commented on [Mar 30, 2018, 5:55 AM GMT+10](https://github.com/nvaccess/nvda/issues/8134#issue-309883479 "2018-03-29T19:55:47Z - Replied by Github Reply Comments"):
> 2. Navigate to the "Search" combo box
> 3. Notice that when attempting to read by character in Browse Mode that this is no longer possible; NVDA treats the control as a single character
> 
> Expected:
> 
> Previously, it was possible to read character-by-character; this seems to now not be possible with several types of controls (including both edit fields and combo boxes).

What characters do you expect to see? Have you typed anything into the box? Your STR doesn't indicate this, so I'm not certain. If there's nothing typed into the box, I wouldn't expect to be able to navigate by character. Note tha tthe label for this box isn't visible on screen anywhere, so it isn't rendered into browse mode as text you can navigate. This is expected and intended behaviour.

**Comment by elliott94** [comment]
If this isn't displayed anywhere, this makes perfect sense - I used to be able to read this character-by-character before updating several months ago, so assumed that this might be a newly-introduced issue. Shall I close?
